WFP-VAM/prism-frontend
# Generate url for analysis and allow users to copy/share

This PR closes #434 
Deployed at : http://prism-434.surge.sh/

To test this PR:
1. REACT_APP_COUNTRY=cambodia yarn start (or pick any other country) or go to http://prism-434.surge.sh/
2. Run analysis: 10-day rainfall + IDPoor, any threshold/date
3. Click 'run analysis', wait for the result
4. Click on "Share analysis", the URL will be copied to clipboard.
5. Open a new tab and paste the URL to the new tab. You can see that the analysis table is prefilled with the same settings. Click "Run analysis" and wait for the result. The result in the new tab should be the same as the old tab,
